src/fwriteR.c

```diff
@@ -5,18 +5,23 @@
 #define DATETIMEAS_EPOCH     2
 #define DATETIMEAS_WRITECSV  3
 
+static bool utf8=false;
+static bool native=false;
+#define TO_UTF8(s) (utf8 && NEED2UTF8(s))
```

Comment:
nit: `TO_UTF8` reads to me like "convert `s` to UTF-8" but IINM it's a logical check. Unfortunately the most logical choice is `NEED2UTF8` which is taken... `REALLY_NEED2UTF8`? :stuck_out_tongue: 

Nothing comes to mind unfortunately, worth considering some more though.